docs/under-the-hood/storage.md

```diff
@@ -115,19 +119,56 @@ Minimal size without values: **2-9B**
 
 ### Marvel dataset use case
 
-Taking into account the Marvel dataset, which has 3 labels and 3 property
-indices, the calculation is as follows:
+The Marvel dataset consists of `Hero`, `Comic` and `ComicSeries` labels, which are indexed. There are also three label-property indices - on the `name` property of `Hero` and `Comic` vertices, and on the `title` property of `ComicSeries` vertices. The `ComicSeries` vertices also have the `publishYear` property.
 
-- Nodes x (`Vertex` + `name` property (let's assume each name is on average 10
-chars long, therefor the name on average has 2B+10B) + `SkipListNode` + `next_pointers` +
-`Delta`) = 21723×(112+12+16+16+104)
-- Edges x (`Edge` + `SkipListNode` + `next_pointers` + `Delta`) = 682943×(40+16+16+104)
-- Label Index Entry x (`SkipListNode<LabelIndex::Entry>` + `next_pointers`) = 3×21723×(24+16)
-- Label Property Index Entry x (`SkipListNode<LabelIndex::Entry>` + `name`
-  property + `next_pointers`) = 3×21723×(80+12+16)
+ <img src={require('../data/under-the-hood/marvel-dataset-schema.png').default}/>
 
-In total **~130MB**, but bear in mind the number can vary because objects
-can have higher overhead due to the additional data.
+There are 6487 `Hero` and 12661 `Comic` vertices with the property `name`. That's 19148 vertices in total. To calculate how much storage those vertices and properties occupy, we are going to use the following formula:
+
+$\texttt{NumberOfVertices} \times (\texttt{Vertex} + \texttt{properties} + \texttt{SkipListNode} + \texttt{next\_pointers} + \texttt{Delta}).$
```

Comment:
I would actually write the formula like this

Vertex + Delta + Skiplist + pointers + properties

BCS that's how we explain them in the text, that's the order of explanation

Reply:
`Vertex` is placed inside `SkipList` structure that has an overhead in the terms of `SkipListNode` and `next_pointers`.  `Vertex` has a pointer to `Delta` object. I think that the current ordering makes more sense, regardless of the order of explanation.